docker-java-api/src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -131,6 +132,8 @@ public interface DockerClient extends Closeable {
      */
     LoadImageCmd loadImageCmd(@Nonnull InputStream imageStream);
 
+    LoadImageAsyncCmd loadImageAsyncCmd(@Nonnull InputStream imageStream);
```

Comment:
Looking at the API, would a logical follow-up be to rename `loadImageCmd` to `loadImageSyncCmd` as well? I see this for the exact method.

Reply:
not renaming now. Maybe create a new method for now and deprecate `loadImageCmd` and then drop it. Although, TBH I think we can avoid it by the using javadocs 🤔